components/organisms/UserSettingsPage/user-settings-page.tsx

```diff
@@ -162,7 +164,7 @@ const UserSettingsPage = ({ user }: userSettingsPageProps) => {
     const payload: UpdateUserPayload = {
       name: formRef.current!.nameInput.value,
       email,
-      bio: formRef.current!.bio.value,
+      bio: bio,
```

Comment:
refactor (blocking): use object property shorthand.

```suggestion
      bio,
```